### DISCUSSION TITLE
Mail forwarding

### DISCUSSION THREAD

**Discussion by rheaditi** [vercel/community#1337] [Help] [answered]
Hi!

Is there a way to set up email forwarding for a custom domain via vercel?
My domain provider is namecheap.com and I'm currently using vercel's NS and CDN.

When my domain was entirely on namecheap, this [email forwarding feature](https://www.namecheap.com/support/knowledgebase/article.aspx/308/2214/how-to-set-up-free-email-forwarding) was available via their dashboard, but it is not anymore since I'm using custom nameservers.

Any help would be appreciated! Thank you :)

**Comment by wiiiimm** [comment, marked as answer]
Hi @rheaditi,

Vercel focuses on hosting solutions and offers services like Nameservers and CDN but we do not have anything email related.

While we do not support email hostings, you are more than welcome to use any 3rd party (or in house) email servers when using the Vercel Nameservers. This is done through the setup of MX and TXT (SPF) records.

Since the free email forwarding solution from Namecheap is only offered to users of Namecheap's Nameservers (DNS services), you might need to look for an alternative (or use their Private Email service). In theory, if you know the MX records used for the email forwarding at step 4 

> 4. Find the Mail Settings section and select Email Forwarding from the drop-down menu. MX records will be set automatically once you Save all changes...

you should be able to continue to use Namecheap's free forwarding service but this very likely to be restricted at their application level.

Alternatively, you can continue to use their Nameservers and point subdomains over to Vercel using the CNAME alias.zeit.co. You will not be able to point your apex domain over since Namecheap does not support ALIAS/ANAME but you can use their no-www to www redirection service to redirect all apex domain traffic to the www subdomain.

**Reply by adrianhorning08** [reply]
@rheaditi did you ever figure out how to do this? Running into the same problem. And this answer doesn't make a lot of sense to me 😅

**Reply by rheaditi** [reply]
Hey @adrianhorning08! I don't recall but I do not have this setup right now, so I don't think I even got it to work. Sorry! 😞

**Reply by amyegan** [reply]
Hi @adrianhorning08 and @rheaditi. You should be able to use a forwarding service, like ImprovMX or Forward Mail, to forward your custom domain emails or use any custom email hosting service you choose. This article walks through some options for using email with a domain using Vercel nameservers https://vercel.com/support/articles/using-email-with-your-vercel-domain

**Comment by titanism** [comment]
See <https://forwardemail.net/guides/vercel>! 🎉
